### PR TITLE
fix: update tailwind integration

### PR DESCRIPTION
**Pull request by kyhyco**
**What changed? Why?**
- update tailwind integration guide

**Notes to reviewers**
To test `tailwind.css` or `styles.css`, you need to run `yarn dev --clean` to run the doc site.

Otherwise, it may use a previous css cache which may give you the wrong result.

**How has it been tested?**
locally

### CODE REVIEW

File: .changeset/curvy-ties-repeat.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"@coinbase/onchainkit": patch
+---
+
+- **docs**: Update tailwind integration guide. By @kyhyco #636
```
Comment (kyhyco) [inline .changeset/curvy-ties-repeat.md:5]: I think we will need this guy [chore: upgrade packemon to v4](https://github.com/coinbase/onchainkit/pull/616) or at least update all the `lib` to `esm` in the main package.json file.

Docs `yarn build` is looking for `lib/index.js`. But the problem is OCK `yarn build` does not produce a `lib` folder at the moment.

Comment (Zizzamia) [reply .changeset/curvy-ties-repeat.md:5]: Ah!

Comment (Zizzamia) [reply .changeset/curvy-ties-repeat.md:5]: You right, we don't produce a `lib` folder.

Comment (Zizzamia) [reply .changeset/curvy-ties-repeat.md:5]: Ok, tomorrow morning (in 7 hours) I am going to give a try merging your PRs.

**Review by Zizzamia — APPROVED** [review]

**Pull request merged**